.github/workflows/test.yml

```
@@ -0,0 +1,138 @@
+name: Test the current PR
+on:
+  repository_dispatch:
+    types:
+      - test-command
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - master     
+
+jobs:
+  configure:
+    name: Preliminary configuration
+    runs-on: ubuntu-latest
+    outputs:
+      commit_ref: ${{ steps.configure.outputs.commit_ref }}
```

```suggestion
      commit-ref: ${{ steps.configure.outputs.commit-ref }}
```
Just for consistency